srcpkgs/epson-inkjet-printer-escpr2/template

```diff
@@ -1,20 +1,20 @@
 # Template file for 'epson-inkjet-printer-escpr2'
 pkgname=epson-inkjet-printer-escpr2
-version=1.1.24
+version=1.1.53
 revision=1
-archs="x86_64 i686 aarch64 armv7l"
+archs="x86_64* i686 aarch64 armv7l"
 create_wrksrc=yes
 build_wrksrc="${pkgname}-${version}"
 build_style=gnu-configure
 configure_args="--with-cupsfilterdir=/usr/lib/cups/filter --with-cupsppddir=/usr/share/ppd"
-hostmakedepends="automake rpmextract "
+hostmakedepends="automake rpmextract"
 makedepends="cups-devel ghostscript-devel"
 short_desc="Epson Inkjet Printer Driver 2 (ESC/P-R) for Linux"
 maintainer="André Cerqueira <acerqueira021@gmail.com>"
 license="custom:Proprietary"
 homepage="http://download.ebz.epson.net/dsc/search/01/search/?OSC=LX"
-distfiles="https://download3.ebz.epson.net/dsc/f/03/00/12/09/63/b7d2bb6a97c9ad99a96ebc68f8abcb1254888e94/${pkgname}-${version}-1lsb3.2.src.rpm"
-checksum=0cfce328ab1359b7c393532c54f5cdb0ee43ec2fd188d7a09b0ab6b04331d9ca
+distfiles="https://download3.ebz.epson.net/dsc/f/03/00/14/04/43/912b0e6f8a4a51379d983f23abc3db087ef20bc6/epson-inkjet-printer-escpr2-${version}-1lsb3.2.src.rpm"
+checksum=e72c5b47a77ef8e952846ca70b18aa88823149bd982d061257be07845d9ad3c6
 restricted=yes
 repository=nonfree
 
```

Comment:
And add `--strip-components=1` to `bsdtar`

Reply:
Fixed